packages/web-components/fast-foundation/src/accordion-item/accordion-item.ts

```diff
@@ -56,12 +56,12 @@ export class AccordionItem extends FoundationElement {
      * HTML Attribute: id
      */
     @attr
-    public id: string;
+    public id: string | undefined;
```

Comment:
For primitives, are we preferring
```suggestion
    public id: string | undefined;
```
or
```suggestion
    public id: string = "";
```